src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
 
+  return totalCost;
```

Comment:
Consider using descriptive constants for the daily cost, long-term rental threshold, and discounts. This will make the code more readable and maintainable. For example, use `const DAILY_COST = 40`, `const LONG_TERM_RENTAL = 7`, `const LONG_TERM_DISCOUNT = 50`, `const SHORT_TERM_RENTAL = 3`, and `const SHORT_TERM_DISCOUNT = 20`.